src/seedu/addressbook/data/person/Street.java

```diff
@@ -0,0 +1,16 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Created by Herman on 27/6/2017.
+ */
+public class Street {
+    private String number;
```

Comment:
Ironically named variable? Also avoid using the same variable name for all classes as it can cause confusion.